tests/gtest/avify4mtest.cc

```diff
@@ -56,6 +56,42 @@ TEST_P(Y4mTest, EncodeDecode) {
   EXPECT_TRUE(testutil::AreImagesEqual(*image, *decoded));
 }
 
+TEST_P(Y4mTest, OutOfRange) {
+  const int width = std::get<0>(GetParam());
+  const int height = std::get<1>(GetParam());
+  const int bit_depth = std::get<2>(GetParam());
+  const avifPixelFormat yuv_format = std::get<3>(GetParam());
+  const avifRange yuv_range = std::get<4>(GetParam());
+  const bool create_alpha = std::get<5>(GetParam());
+  std::ostringstream file_path;
+  file_path << testing::TempDir() << "avify4mtest_outofrange_" << width << "_"
+            << height << "_" << bit_depth << "_" << yuv_format << "_"
+            << yuv_range << "_" << create_alpha;
+
+  testutil::AvifImagePtr image = testutil::CreateImage(
+      width, height, bit_depth, yuv_format,
+      create_alpha ? AVIF_PLANES_ALL : AVIF_PLANES_YUV, yuv_range);
+  ASSERT_NE(image, nullptr);
+
+  // Insert out-of-range values on purpose if possible.
+  const uint32_t yuva8[] = {255, 0, 255, 255};
```

Comment:
To the new version of `y4mClampSamples()`, the values 0 and 255 are no longer out of range for bit depth 8. So this is confusing for people who don't know the history of `y4mClampSamples()`.

Similarly, in the next line the value 0 is no longer out of range for bit depths 10 and 12.

This can be addressed by clarifying the comment (i.e., saying that some values are out of range for limited range) or by modifying the values in these two arrays.